README.md

```diff
@@ -284,15 +284,20 @@ can be bundled or downloaded on first launch via `pip`.
 ### Shiv ###
 
 `shiv` allows you to bundle zipapps with C extensions, but doesn't provide for any `online` installs
-and will extract everything into one `~/.shiv` directory unless otherwise specified. 
-`ducktools-env` will create a separate environment for each unique set of requirements
-for temporary environments by matching specification.
+and will extract everything into one `~/.shiv` directory unless otherwise specified.
+At the time of writing support for using inline script metadata has not yet been merged but there
+is a PR to add support.
 
-### PEX ###
+`ducktools-env` creates and manages virtual environments for each unique set of script requirements.
+These are kept in more platform specific directories documented earlier in the readme.
 
-`pex` provides an assortment of related tools for developers alongside a `.pex` bundler.
-It doesn't (to my knowledge) have support for inline script metadata and it makes `.pex` files
-instead of `.pyz` files.
+### Pex ###
+
+`Pex` provides an assortment of related tools for developers alongside a `.pex` bundler.
+It has (undocumented) support for inline script metadata for building its archives and will
```

Comment:
I think this is fine to say, and Pex documentation is non-existent, but I'll also point out that all Pex features are documented via --help. I acknowledge most people don't like to read that or don't find it useful.

Reply:
I don't really think of `--help` as documentation as such, it's not so much that it's not useful but that it's only useful when you've already decided to use the tool. If I didn't know 'Pex' supported inline metadata because you've told me, I wouldn't install it in order to find out, so I'd never look at the `--help` information.

Reply:
Ack. I'm just 1 guy and can only do so much. Docs will continue to have to wait until I get a few more things in good shape.

Reply:
Completely understandable wrt docs being outdated. This isn't intended as a dig.

The feature I actually use most of the time in `ducktools-env` isn't the bundling at all, but the script registration feature for scripts I don't want to put on `PATH` but do want access to. But that wasn't the original plan when I wrote the readme so it's sort of mentioned off hand halfway through.